install/streams.csv

```diff
@@ -1129,7 +1129,8 @@ KRVZ,10,E,H,100,false,false,false,2004-12-03T22:00:00Z,9999-01-01T00:00:00Z
 KRVZ,30,L,D,1,false,false,false,2012-11-22T20:30:00Z,9999-01-01T00:00:00Z
 KRVZ,30,H,D,100,false,false,false,2012-11-22T20:30:00Z,9999-01-01T00:00:00Z
 KTGO,10,E,H,100,false,false,false,2014-03-15T01:41:00Z,2014-04-23T23:25:00Z
-KUTZ,10,E,H,100,false,false,false,2010-04-15T02:00:00Z,9999-01-01T00:00:00Z
+KUTZ,10,E,H,100,false,false,false,2010-04-15T02:00:00Z,2024-06-17T21:32:00Z
+KUTZ,10,H,H,100,false,false,false,2024-06-17T22:08:00Z,9999-01-01T00:00:00Z
```

Comment:
New streams are still EHZ|N|E (Lennartz are short period sensors) so no need for the new line?

Reply:
Oops, I just saw short period sensor and jumped immediately to a BB upgrade. Fixed now, though I should have seen that in the streams.